cpuinfo.go

```diff
@@ -207,49 +190,53 @@ func parseCPUInfoARM(info []byte) ([]CPUInfo, error) {
 		}
 		field := strings.SplitN(line, ": ", 2)
 		switch strings.TrimSpace(field[0]) {
+		case "Processor":
+			// this appears at the first line in kernels < 3.7
+			modelName = field[1]
 		case "processor":
-			cpuinfo = append(cpuinfo, commonCPUInfo) // start of the next processor
+			cpuinfo = append(cpuinfo, CPUInfo{VendorID: modelName}) // start of the next processor
```

Comment:
See this comment on a similar current PR: https://github.com/prometheus/procfs/pull/296/files#discussion_r427036662

Reply:
Thanks, fixed!

Reply:
The link appears to be not as helpful as I wanted, but the conversation was about that "processor" is only present for CONFIG_SMP (pre v3.8) kernels, so you likely need to figure out how to support kernels without that option.

Reply:
Yeah, I think I have figured it out, please see the updated commit 928d8ea1fea240, to which I have added a test case with data from the above comment.

Reply:
👍 Cool!

Reply:
@bluecmd can you review it then? I am aware you're not a maintainer but you wrote some of this code.

Reply:
I didn't write any of this code, I just filed bugs against it :-)